src/test/java/org/jabref/logic/util/FileNameCleanerTest.java

```diff
@@ -12,9 +12,9 @@ class FileNameCleanerTest {
     @ParameterizedTest
     @CsvSource({
         "legalFilename.txt, legalFilename.txt",
-        "illegalFilename/?*<>|.txt, illegalFilename______.txt",
-        "illegalFileName{.txt, illegalFileName_.txt",
-        "?The Evolution of Sentiment} Analysis}.PDF, _The Evolution of Sentiment_ Analysis_.PDF"
+        "illegalFilename______.txt, illegalFilename/?*<>|.txt",
```

Comment:
This was right before, the first argument in csv source would be the expected argument, aka the ground truth

Reply:
Its OK now!

```diff
-  assertEquals("illegalFilename______.txt", FileNameCleaner.cleanFileName("illegalFilename/?*<>|.txt"));
+  "illegalFilename______.txt, illegalFilename/?*<>|.txt",
```

Reply:
Ah yes, sorry tired